### ISSUE TITLE
Panther Router and Factory Addess Error

### ISSUE THREAD

**Issue by rodolfopatane** [Haehnchen/uniswap-arbitrage-flash-swap#2]
What you say panther is really Jetswap.

This documentation of Panther Swap:
https://docs.pantherswap.com/products/amm#contracts

Factory: 0x670f55c6284c629c23bae99f585e3f17e8b9fc31
Router: 0x24f7c33ae5f77e2a9eceed7ea858b4ca2fa1b7ec

**Comment by mindfulme** [comment]
be more specific, is there only a name typo? it is not a problem at all

**Comment by rodolfopatane** [comment]
> be more specific, is there only a name typo? it is not a problem at all

In addresses file the constant name say 'panther', but the contract address is the Jetawap Exchange not Phanter Exchange. is not a type error, is a reference error.

**Comment by mindfulme** [comment]
yeah, feel free to rename it. I have added about 15 more DEXs to my config with routers and factories
